Week2/alaa.js

```diff
@@ -0,0 +1,97 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
+//3
+var x;
+console.log('the value of my variable x will be: undefined');
+x = 7; // initialize the value of x
+console.log('the value of my variable x will be 7');
+console.log(x);
+//4
+var y;
+y = 'hyf';
```

Comment:
Two things here please:
1. You can use `let` keyword here instead of `var`.
2. You can simply combine the two statements together i.e do `let y = 'hyf'`;